src/scripts/public/ribbon-navigation.js

```diff
@@ -166,7 +159,7 @@ const domReady = () => {
                             class: 'wp-block-navigation__submenu-container',
                         });
                         currentUl.appendChild(ul3);
-                        currentUl = ul3;
+                        currentUl = ul3; //?
```

Comment:
I'm guessing this comment doesn't need to be here

Reply:
Correct. Will remove.